hpeOneView/oneview_client.py

```diff
@@ -974,9 +974,10 @@ def labels(self):
         Returns:
             Labels:
         """
-        if not self.__labels:
+        '''if not self.__labels:
```

Comment:
you can remove these commented lines (977 to 979)

Reply:
done